defaults/main.yml

```diff
@@ -8,3 +8,6 @@ k8s_letsencrypt_email: ''
 k8s_echotest_state: present
 k8s_echotest_hostname: 'echotest.caktus-built.com'
 k8s_echotest_letsencrypt_issuer: 'letsencrypt-staging'
+
+k8s_papertrail_logspout_namespace: 'default'
+k8s_papertrail_logspout_destination: ''
```

Comment:
Not sure if we are providing documentation hints here, or elsewhere, but might be nice to say that this should look like `syslog+tls://logsN.papertrailapp.com:XXXXX`

Reply:
Good idea!